packages/atomic/src/utils/dom-utils.tsx

```diff
@@ -6,3 +6,45 @@ export function rectEquals(r1: DOMRect, r2: DOMRect) {
     r1.height === r2.height
   );
 }
+
+/**
+ * Returns the padding values of an element.
+ */
+export function getElementPadding(element: HTMLElement) {
+  const styles = window.getComputedStyle(element);
+
+  return {
+    top: parseFloat(styles.paddingTop.length ? styles.paddingTop : '0'),
+    right: parseFloat(styles.paddingRight.length ? styles.paddingRight : '0'),
+    bottom: parseFloat(
+      styles.paddingBottom.length ? styles.paddingBottom : '0'
+    ),
+    left: parseFloat(styles.paddingLeft.length ? styles.paddingLeft : '0'),
+  };
+}
+
+/**
+ * Returns the absolute width of an element.
```

Comment:
```suggestion
 * Returns the absolute height of an element.
```

Reply:
removed those utils as I'm not using them in the end but good catch 👍